opm/simulators/wells/WellGroupHelpers.hpp

```diff
@@ -446,6 +496,48 @@ namespace Opm {
         return;
     }
 
+    inline double groupFractionFromPotentials(const Group& group, const Schedule& schedule, const WellStateFullyImplicitBlackoil& wellState, const int reportStepIdx, const int phasePos, const bool isInjector) {
+        double groupTotalGuideRate = 0.0;
+        const Group& groupParent = schedule.getGroup(group.parent(), reportStepIdx);
+        for (const std::string& groupName : groupParent.groups()) {
```

Comment:
Use "siblingName" rather than "groupName" for clarity, "siblingGroup" rather than "groupTmp".